package.json

```diff
@@ -8,7 +8,7 @@
     "dev": "nodemon",
     "start": "probot run ./index.js",
     "test": "npm-run-all --print-label --parallel lint:* --parallel test:*",
-    "lint:js": "standard",
+    "lint:js": "prettier-standard --format --lint",
```

Comment:
i usually prefer an approach where the lint scripts only verify and do not automatically apply formatting. that way the fix/format step is a little more intentional and the lint script can run the same way with a manual run of `npm test` as it does on CI. would it make sense to move the auto-format piece to `lint:js:fix`?

Reply:
Sure, we can do that.

FWIW, I am used to having the auto-format [run on save in my editor](https://marketplace.visualstudio.com/items?itemName=numso.prettier-standard-vscode).

Reply:
Resolved in b28583ff5b4e05f5a23de57cfcdc885a2b8cfd4f